chained mutations explanation (#206)
AdobeDocs/graphql-mesh-gateway: src/pages/gateway/work-with-mesh.md

  }
}
```

## Chained mutations

> When you query a mesh using chained mutations, it will cause multiple mutations calls to the source instead of one single call. This happens because GraphQL requests execute when the GraphQL server receives them, instead of proxying the calls. Because API Mesh supports multiple sources, it has to evaluate batched mutations and separate the information based on the source it is associated with.

Reviewer: ```suggestion
When you query a mesh using chained mutations, the mesh makes a separate call to the source for each mutation.  This happens because GraphQL requests execute when the GraphQL server receives them, instead of proxying the calls. Because API Mesh supports multiple sources, it evaluates batched mutations and separates the information based on the source it is associated with.
```
